submissions/xT0PERx/html-movie-seat-booking/main.html

```diff
@@ -0,0 +1,249 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="main.css" />
+    <title>Movie Seat Booking</title>
+  </head>
+  <body class="page">
+    <header class="top">
+      <nav class="navigation">
+        <ul class="items">
+          <li class="item"><a href="#">Main</a></li>
+          <li class="item"><a href="#">Premieres</a></li>
+          <li class="item"><a href="#">Contacts</a></li>
+          <li class="item navigation-item-choisen">
+            <a href="#">Booking tickets</a>
+          </li>
+        </ul>
```

Comment:
Avoid using too general/abstract class names. 
```suggestion
        <ul class="navigation-items">
          <li class="navigation-item"><a href="#">Main</a></li>
          <li class="navigation-item"><a href="#">Premieres</a></li>
          <li class="navigation-item"><a href="#">Contacts</a></li>
          <li class="navigation-item navigation-item-choisen">
            <a href="#">Booking tickets</a>
          </li>
        </ul>
```